CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -209,27 +207,22 @@ public function buildQuickForm() {
       return;
     }
 
-    CRM_Core_Payment_Form::buildPaymentForm($this, $this->_paymentProcessor, FALSE, TRUE, CRM_Utils_Request::retrieve('payment_instrument_id', 'Integer'));
+    if ($this->_mode) {
```

Comment:
I think with changes since merged in core this if ($this->_mode should not be required - see https://github.com/civicrm/civicrm-core/pull/10157 )